docs/tutorial/using-checkbox/manifest.rst

```diff
@@ -0,0 +1,299 @@
+select ``Yes``, then press ``T`` to start testing.
+
+The jobs in the test plan will be executed, the test session will wrap up and
+Checkbox will ask if you want to upload your results. Select ``n`` and press
+``Enter`` to finish.
+
+Let's look at what happened by looking at the text on the screen. Towards
+the top, we can see:
+
+.. code-block:: none
+
+    =========================[ Resume Incomplete Session ]==========================
+    There are 0 incomplete sessions that might be resumed
+    Preparing...
+    Saving manifest to /var/tmp/checkbox-ng/machine-manifest.json
+
+The manifest file is a JSON file that Checkbox stores at a specific
+location. If this file does not exist, it is created. It is then updated
+using the values chosen in the System Manifest screen.
+
+A bit below, we see a ``Hardware Manifest`` job being executed:
+
+.. code-block:: none
+    :emphasize-lines: 11
+
+    =========[ Running job 2 / 3. Estimated time left (at least): 0:00:02 ]=========
+    -----------------------------[ Hardware Manifest ]------------------------------
+    ID: com.canonical.plainbox::manifest
+    Category: com.canonical.plainbox::uncategorised
+    ... 8< -------------------------------------------------------------------------
+    ns: com.canonical.certification
+    name: checkbox-provider-base
+    has_audio_playback: 
+    (...)
+    has_touchpad: 
+    has_touchscreen: True
+    (...)
+    has_wlan_adapter: 
+    has_wwan_module:
+     
+    (...)
+    
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job passed
+
+This job will collect the information from the manifest file so that it
+can be used by Checkbox later for each :ref:`manifest_entry` defined in the
+providers. We can see that ``has_touchscreen`` (the :ref:`id<Manifest Entry
+id field>` of the manifest entry unit that represents whether or not this
+device has a touch screen) is set to ``True`` because we selected it in the
+System Manifest screen.
+
+Finally, a job that uses this information, ``tutorial/manifest``, is executed:
+
+
+.. code-block:: none
+
+    =========[ Running job 3 / 3. Estimated time left (at least): 0:00:01 ]=========
+    ---------------------------[ A job using a manifest ]---------------------------
+    ID: com.canonical.certification::tutorial/manifest
+    Category: com.canonical.certification::tutorial
+    ... 8< -------------------------------------------------------------------------
+    This test is executed because user said this device has a touchscreen.
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job passed
+
+Modifying the manifest information within Checkbox
+==================================================
+
+Let's run the same test plan again, but this time we will pretend the device
+has no touch screen.
+
+- Launch Checkbox using ``checkbox.checkbox-cli``.
```

Comment:
Since these steps are to be executed sequentially, it's better to use a numbered list instead of bullets

Reply:
Fixed in [dfd5bf2](https://github.com/canonical/checkbox/pull/736/commits/dfd5bf24e3c0afde27e0d9997c9df82c766010d3)